cmd/machine-config-daemon/pivot.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // flag storage
 var keep bool
-var reboot bool
 var fromEtcPullSpec bool
 
 const (
-	etcPivotFile       = "/etc/pivot/image-pullspec"
-	runPivotRebootFile = "/run/pivot/reboot-needed"
+	// etcPivotFile is used for 4.1 bootimages and is how the MCD
+	// currently communicated with this service.
```

Comment:
nit: Unclear as to if this means now or previously. Currently would indicate it's in use now while communicated would indicate the past.

Reply:
It was noted this is a direct pull in from https://github.com/openshift/machine-config-operator/pull/1809/files#r442404093